app/views/conferences/location.html.erb

```diff
@@ -20,7 +20,7 @@
           <div class="contact-det">
             <p><b><%= contact.name %> <%= contact.surname %></b><p>
             <td><% contact.splited_mail.each do |mail| %>
-              <p><b>Mail: </b><%= mail %><br></p>
+              <p><b>Mail: </b><a href="mailto:<%= mail %>"><%= mail %></a><br></p>
```

Comment:
we can use rails form helpers http://apidock.com/rails/ActionView/Helpers/UrlHelper/mail_to